### PR TITLE
Table DDL product tests

### PR DESCRIPTION
**Pull request by cawallin**
- CREATE TABLE, CREATE TABLE IF NOT EXISTS
- DROP VIEW IF EXISTS
- DROP TABLE IF EXISTS

### CODE REVIEW

**Comment by cawallin** [comment]
@kokosing or someone else familiar with tempto, can you review this?

File: presto-product-tests/src/test/java/com/facebook/presto/tests/CreateDropTableTests.java [hunk 1]
```diff
@@ -48,4 +48,35 @@ public void shouldCreateTableAsEmptySelect()
...
+    public void shouldCreateTableIfNotExists()
+            throws Exception
+    {
+        String tableName = "nation";
+        Table table = createPrestoTable(tableName, "CREATE TABLE IF NOT EXISTS %s (n_nationkey BIGINT, n_name VARCHAR, n_regionkey BIGINT, n_comment VARCHAR )");
+        assertThat(query(format("SELECT * FROM %s", table.getNameInDatabase()))).hasRowsCount(25);
+    }
+
+    @Test(groups = CREATE_DROP_TABLE)
+    public void shouldDropTableIfNotExists()
```
Comment (rschlussel-zz) [inline presto-product-tests/src/test/java/com/facebook/presto/tests/CreateDropTableTests.java:74]: shouldDropTableIfExists

**Comment by rschlussel-zz** [comment]
Maybe add tests that create table if not exists works as usual if the table does not exist, or that it gets an error message if the table exists and you don't have if not exists specified.  Likewise for drop table.  I don't know how thorough we're trying to be.

**Comment by cawallin** [comment]
@rschlussel I agree we should do more testing if we're trying to be thorough, but I'm going to just push these tests so they get into 1.5 and we can write more later.

**Pull request merged**